packages/KTable/KTable.vue

```diff
@@ -184,6 +217,19 @@ export default {
 
 table.k-table {
   border-collapse: collapse;
+
+  &.side-border {
+    border-collapse: separate;
+    border-spacing: 0 5px;
+
+    tr:hover {
+      background-color: var(--KTableHover, var(--steal-100, color(steal-100)));
+    }
+
+    td:first-child {
+      border-left: 3px solid var(--KTableBorder, var(--steal-200, color(steal-200)));
```

Comment:
in stead of adding to the first td, lets instead set border: none/0 to the whole TR and then use a box shadow here as its a little more flexible and also won't eat into the content. It should also be 2px & steal-300 per the design. Lets also remove this hover declaration and use steal-100 for the `.has-hover` class

```css
tr {
  border: none;
  box-shadow: -2px 0 0 var(--KTableBorder, var(--steal-300, color(steal-300)));
}
```

Reply:
I defer to the CSS master here